docs/lago-tutorial/README.md

```diff
@@ -0,0 +1,206 @@
+
+//TODO: elaborate about the different features and values that can be specified within the file.
+
+#### Creating The Environment
+
+
+We can use two different approches:
+
+Manual Configurations:
+
+```
+lago init --template-repo-path=../../templates-repo/template-repo.json lago-work-dir init.json
+```
+(This should be invoked from: /lago-tutorial/scenarios/jenkins)
+
+* The directory /lago-work-dir will contain the files of our new Lago environment.
+This directory shouldn't exist before invoking lago init.
+* --template-repo-path, for specifing the path to the template-repo file.
+* init.json, the name of the file which describes our environment.
+
+Auto Configurations
+
+```
+lago init
+```
+This command will use the following default configurations:
+* The command will serch for a file named "LagoInitFile" that will be used as the init file.
+* The workdir will be named ".lago"
+* The [http://templates.ovirt.org/repo/repo.metadata](default template repo file) will be used.
+
+Note: by default the init.json file is configured with a relative path from /lago-work-dir to the deployment scripts.
+If you are using the Auto configuration, or give any other name to the workdir, you should edit the relative path.
+
+#### Deploy the VMs
+
+When using the manual configurations all the commands should be invoked from /lago-work-dir.
+
+When using the default workdir you can invoke commands from it's parent, or any 'sibiling', for example:
+```
+lago init
+mkdir lolo
+cd lolo
+lago status
+```
+
+Now, lets start the vms:
+
+```
+lago start
+```
+Or for a specific vm named "server":
+
+```
+lago start server
+```
+
+The command below will run the deployment scripts (from within the vms) that were specified in the init.json file.
+This command should be invoked from the directory that will satisfy the relative path to the deployment scripts,
+as mentiod in the init.json file.
+
+so, if you used the "manual configurations", the command should be invoked from /lago-work-dir.
+
+```
+lago deploy
+```
+Jenkins will be installed on the server.
+OpenJDK will be installed on the slaves.
+
+### Getting the state of the environment
+
+You cae get information about the state of the enviorment with:
+
+```
+lago status
+```
+Or for a formated output as json or yaml:
+
+```
+lago --out-format [json | yaml] status
+```
+
+You can write down to yourself the ip adresses of the server and slaves,
+as we will need them when configuring the server.
+
+### Interacting with the VMs
+
+Lago allows you to connect to the vms via ssh.
+for exmaple, if we have a vm named "server" we will use the following:
+
+```
+lago shell server
+```
+If the deployment scripts run successfuly we don't have 
+to connect to the machines.
+
+In case of a failure, you can access the vms via console.
+This is useful when the vm failed to boot or when trubleshooting network issues.
+
+```
+lago console server
+```
+
+### Adding the Jenkins slaves
```

Comment:
This one is the one that is mostly scenario-specific, I suggest leaving the long start/stop/deploy... explanations in this doc, and then, on the jenkins scenario, just use them right away without explaining too much